<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgot_password</class>
 <widget class="QDialog" name="Forgot_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>647</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>647</width>
    <height>673</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Librarian - Forgot Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Forgot_password
{
	border-image: url(:/img/images/bg_forgot.png);
}

#frame_fPass
{
background: rgba(0,0,0,0.8);
border-radius: 20px;
}

#lineEdit_username
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

#lineEdit_NewPassword
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

#lineEdit_ConfirmPassword
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

#lineEdit_answer
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

#comboBox_question
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}


QFrame
{
background: rgba(0,0,0,0.8);
border-radius: 20px;
}


QLineEdit
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 161, 29, 255), stop:0.134831 rgba(255, 110, 62, 255), stop:0.230337 rgba(255, 81, 105, 255), stop:0.342697 rgba(255, 28, 174, 255), stop:0.466292 rgba(193, 0, 197, 255), stop:0.589888 rgba(212, 0, 255, 255), stop:0.764045 rgba(171, 14, 223, 255), stop:0.955056 rgba(101, 14, 223, 255));
border-radius: 60px;

}

QLabel
{
	
	
	color: rgb(237, 174, 254);
background: transparent;

}

QPushButton
{
color: white;
background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0.00561798 rgba(69, 0, 103, 255), stop:0.134831 rgba(87, 0, 129, 255), stop:0.297753 rgba(110, 0, 165, 255), stop:0.550562 rgba(117, 0, 117, 255), stop:0.674157 rgba(124, 0, 124, 255), stop:0.837079 rgba(140, 0, 140, 255), stop:0.977528 rgba(172, 0, 172, 255));
border-radius: 15px;

}

QPushButton:hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.494, stop:0 rgba(244, 86, 139, 255), stop:0.393258 rgba(255, 0, 255, 255), stop:0.758427 rgba(162, 21, 232, 255), stop:1 rgba(69, 0, 103, 255));
color: black;
}

QCheckBox
{
;
	color: rgb(170, 170, 255);
}

QComboBox
{
background: transparent;
border: none;
color: #717072;
border-bottom : 1px solid #717072;
}

QComboBox QAbstractItemView::item 
{ 
height: 40px;
color: #717072;
background: rgb(38,38,38);
} 
QComboBox QAbstractItemView::item:selected 
{ 
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0.994318, stop:0 rgba(255, 103, 156, 255), stop:0.297753 rgba(255, 0, 255, 255), stop:0.646067 rgba(171, 14, 223, 255), stop:1 rgba(85, 0, 138, 255));
color: white;
} 

QMessageBox
{
background-image:url(:/img/images/messagebox.jpg);
font: 9pt &quot;Century Gothic&quot;;
color:white;
}

QMessageBox QPushButton
{	
color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 81, 105, 255), stop:0.303371 rgba(255, 28, 174, 255), stop:0.608939 rgba(193, 0, 197, 255), stop:0.787709 rgba(212, 0, 255, 255), stop:0.966292 rgba(171, 14, 223, 255));
font: 12pt &quot;Century Gothic&quot;;
background-color: transparent;
border: 1px solid qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 81, 105, 255), stop:0.966292 rgba(171, 14, 223, 255));
border-radius: 10px;
height: 20px;
width: 50px;
}

QMessageBox QPushButton:hover
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgba(255, 81, 105, 255), stop:0.303371 rgba(255, 28, 174, 255), stop:0.608939 rgba(193, 0, 197, 255), stop:0.787709 rgba(212, 0, 255, 255), stop:0.966292 rgba(171, 14, 223, 255));
color:white;
font: 12pt &quot;Century Gothic&quot;;
}
QMessageBox QLabel
{
color:white;
background:transparent;
}
</string>
  </property>
  <widget class="QFrame" name="frame_fPass">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>451</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>451</width>
     <height>541</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_fPass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Having Trouble Logging In?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>UID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_answer">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_question">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Select the Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Submit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Cancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NewPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ConfirmPassword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangePass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_admin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>Admin Account</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/img/images/admin-with-cogwheels.png</normaloff>:/img/images/admin-with-cogwheels.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_passStrength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>381</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton_fPass_icon">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/img/images/lock.png</normaloff>:/img/images/lock.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
